components/freertos/include/freertos/task.h

```diff
@@ -181,6 +181,7 @@ typedef struct xTASK_STATUS
 	uint32_t ulRunTimeCounter;		/*!< The total run time allocated to the task so far, as defined by the run time stats clock.  See http://www.freertos.org/rtos-run-time-stats.html.  Only valid when configGENERATE_RUN_TIME_STATS is defined as 1 in FreeRTOSConfig.h. */
 	StackType_t *pxStackBase;		/*!< Points to the lowest address of the task's stack area. */
 	uint32_t usStackHighWaterMark;	/*!< The minimum amount of stack space that has remained for the task since the task was created.  The closer this value is to zero the closer the task has come to overflowing its stack. */
+	BaseType_t xCoreID;
```

Comment:
This line needs a comment similar to the lines above

Reply:
After looking BaseType_t definition in portmacro.h, BaseType_t type is int. I think uint32_t is more suitable for xCoreID.
```
#define portBASE_TYPE	int
typedef portBASE_TYPE			BaseType_t;
```

Reply:
No, I have taken made xCoreID have type BaseType_t because this is how the value is typed in the tskTaskControlBlock (tasks.c) where the value is sourced from.

Reply:
Comment now added